Classes/Console/Command/Database/ImportCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+namespace Helhum\Typo3Console\Command\Database;
+
+/*
+ * This file is part of the TYPO3 Console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Database\Configuration\ConnectionConfiguration;
+use Helhum\Typo3Console\Database\Process\MysqlCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
+
+class ImportCommand extends Command
+{
+    /**
+     * @var ConnectionConfiguration
+     */
+    protected $connectionConfiguration;
+
+    public function __construct(string $name = null, ConnectionConfiguration $connectionConfiguration = null)
+    {
+        parent::__construct($name);
+
+        $objectManager = GeneralUtility::makeInstance(ObjectManager::class);
+        $this->connectionConfiguration = $connectionConfiguration
+            ?? $objectManager->get(ConnectionConfiguration::class);
```

Comment:
using the object manager here is unnecessary and counter productive.

please keep using new, like in `DatabaseExportCommand`